cmd/wit-bindgen-go/cmd/generate/generate.go

```diff
@@ -52,6 +52,13 @@ var Command = &cli.Command{
 			Config:   cli.StringConfig{TrimSpace: true},
 			Usage:    "Import path for the Component Model utility package, e.g. github.com/bytecodealliance/wasm-tools-go/cm",
 		},
+		&cli.StringFlag{
+			Name:     "extern-tag",
+			Value:    "",
+			OnlyOnce: true,
+			Config:   cli.StringConfig{TrimSpace: true},
+			Usage:    "Build tag for wasmimport/wasmexport functions. See https://pkg.go.dev/go/build",
+		},
```

Comment:
Let’s omit this. The build tags can be simple now: `//go:build wasm || wasm32 || tinygo.wasm`.